build.gradle

```diff
@@ -35,25 +35,16 @@ dependencies {
     compile 'com.sun.jersey:jersey-server:1.19.4'
 }
 
```

Comment:
The publishing section is needed so I can call `./gradlew publishToMavenLocal` to generate maven artifacts locally. By all means use the properties ${mavenMajorVersion} etc. By the way in the dev branch gradle.properties should have `majorPatchVersion=0-SNAPSHOT`. It should only get the release version in `master` branch (if I have your workflow correct).

Reply:
Added publishing section back in

Reply:
Thanks. You might not be publishing snapshots or nightlies but any dev should be able to rebuild the release jar at the specificed version locally from whatever branch or tag without manual modification of build.gradle. For this reason the SNAPSHOT belongs in gradle.properties.